packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -943,6 +1009,15 @@ export class Vector3 {
         return Quaternion.RotationYawPitchRoll(this._y, this._x, this._z);
     }
 
+	/**
+	 * Converts the current Vector3 to its polar coordinates
+	 * @returns the current polar coordinates (r, theta, phi)
+	 */
+	public toPolar(): Vector3{

```

Comment:
Same comment than for `Vector2` => add a `toPolarToRef(result: Vector3)` function.

More generally, for all new functions added, you should also add a `xxxToRef` function that takes as the last parameter the result object (see also `getRotationToVector` / `toSpherical` / `fromSpherical` / `RotationBetweenVectors`).